client/src/pages/dashboard/NewEventType.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Container from '../../components/Layout/Container';
+import Grid from '@material-ui/core/Grid';
+import Typography from '../../components/DataDisplay/Typography';
+import Button from '../../components/Inputs/Button';
+import TextField from '@material-ui/core/TextField';
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormControl from '@material-ui/core/FormControl';
+import FormLabel from '@material-ui/core/FormLabel';
+
+const NewEventType = () => {
+    return (
+        <Container>
+            <Grid container justify="space-between" alignItems="center" spacing={2} >
+                <Grid item >
```

Comment:
to make it more mobile friendly, you can probably extend this to make it more mobile friendly

```suggestion
                <Grid item xs="12" md="4">
```